lib/fastly_nsq/version.rb

```diff
@@ -1,3 +1,3 @@
 module FastlyNsq
-  VERSION = '0.6.0'.freeze
+  VERSION = '0.7.0'.freeze
```

Comment:
@alieander I think you're trying to apply a rule from [SemVer](http://semver.org/). At least, I'm assuming -- I don't see where we've explicitly committed the project to using SemVer (there are differences among versions, things that aren't quite SemVer but look like it, etc).

My understanding of SemVer is that the rule you seem to be trying to apply does not apply (yet):

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

And

> Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.

It _could_ be that you are suggesting we're ready for v1.0, which I would rather strongly disagree with.

Can you elaborate a bit on what you want here?